### ISSUE TITLE
Support for Gnome 45

### ISSUE THREAD

**Issue by Kerwood** [koolskateguy89/gnome-shell-extension-spotify-controller#27]
Please add support for Gnome 45

**Comment by SteamWind** [comment]
This extension seems abandoned sadly... No release since 2022

**Comment by koolskateguy89** [comment]
@SteamWind not really, I just have other priorities that take precedence. Would be nice if you could contribute Gnome 45 support 👍

**Comment by Kerwood** [comment]
Have a look at this extension instead.
https://github.com/sakithb/media-controls

**Comment by SteamWind** [comment]
Oh I'm sorry but the media controls extensions seems more interesting. Thanks for the work anyway!
